### PR TITLE
New v63004/gtm3146 subtest (tests GTM-3146 in V63004) 

### PR DESCRIPTION
**Pull request by JakeYottaDB**
Tests the following release note                                                                                                                                                      
                                                                                                                                                                                          
GTM ensures that alias and path settings do not interfere with the intentions of the actions 
it takes using invocations of the POSIX system() function. This issue was only observed in 
the GTM development environment, and was never reported by a user. (GTM-3146)

### CODE REVIEW

File: v63004/outref/gtm3146.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Testing 
+# Create a 1 region DB region DEFAULT
+
+# Create a file, shell.csh, to be sourced by child shells : 
+
+cat shell.csh
+#!/usr/local/bin/tcsh -f
+set echo ; set verbose
+alias cp "jakewashere"
```
Comment (nars1) [inline v63004/outref/gtm3146.txt]: Add a comment here that the MUPIP BACKUP done later (which internally does a "cp") used to fail in prior versions due to the "cp" alias here.

File: v63004/u_inref/gtm3146.csh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+echo 'cat shell.csh'
+cat ./shell.csh
+echo ''
+
+echo '# Set $SHELL env var to have child shell shell.csh '
+setenv SHELL "$PWD/shell.csh"
+echo ''
+
+echo "# Running Mupip with "'$SHELL'" ./shell.csh"
+$MUPIP BACKUP "*" tstBackup >>& backup.log
```
Comment (nars1) [inline v63004/u_inref/gtm3146.csh]: Check $status after MUPIP BACKUP to ensure it is 0. If not, print an error.

File: v63004/outref/gtm3146.txt (hunk 1 above)
Comment (JakeYottaDB) [reply v63004/outref/gtm3146.txt]: The comment has been added to the generated shell.csh file. This way the comment will appear right below the cp alias when shell.csh is cat-ed to the screen.

File: v63004/u_inref/gtm3146.csh (hunk 2 above)
Comment (JakeYottaDB) [reply v63004/u_inref/gtm3146.csh]: A $status check block, mirroring those used after dbcreate and dbcheck, has been added after the $MUPIP BACKUP call.

**Pull request merged**